src/test/java/org/jitsi/meet/test/VideoLayoutTest.java

```diff
@@ -54,63 +54,55 @@ public void setupClass()
     @Test
     public void testVideoLayout()
     {
-        driverVideoLayoutTest(getParticipant1());
+        driverVideoLayoutTest(getParticipant1(), false);
     }
 
     /**
      * The webdriver to test.
      * @param participant to test.
+     * @param isScreenSharing <tt>true</tt> if SS is started and <tt>false</tt>
+     *                        otherwise.
      */
-    void driverVideoLayoutTest(Participant participant)
+    void driverVideoLayoutTest(Participant participant, boolean isScreenSharing)
```

Comment:
I think this method is now superfluous (and the doc needs an update).

Reply:
The docs are updated! I decided to leave it because in future we may want to add more stuff in it.

Reply:
> The docs are updated!

I don't think this actually made it to the PR: https://github.com/jitsi/jitsi-meet-torture/blob/925dec78ff39b10cfbe96e2c56b374ee693e63bd/src/test/java/org/jitsi/meet/test/VideoLayoutTest.java#L61 (this commit is the current head of master)